assets/src/styles/index.scss

```diff
@@ -1,5 +1,5 @@
-.hm-post-history {
-	--post-history-red: var( --post-history-red, #ff424a );
+.post-history {
+	--post-history-red: #ff424a;
```

Comment:
The purpose of this rule was to allow this to be overridden by a consuming project -- why was the fallback removed?

Reply:
The CSS var was not getting defined in the browser and not rendering. It seems that you can't name a var the same as the value like `--post-history-red: var( --post-history-red, #ff424a )`.

I instead renamed it something different to allow for `--post-history-red` to be defined somewhere else and thus overwritten:

```
--post-history-accent: var( --post-history-red, #ff424a );
```